.github/workflows/ci-cd-api-server.yaml

```
@@ -52,26 +53,58 @@
+      # 7. 안전한 배포를 위한 정리 작업 (t3.micro 환경용)
+      - name: Prepare for deployment
         run: |
-          jq '.containerDefinitions[0].image = "${{ env.ECR_REGISTRY }}/${{ env.ECR_REPOSITORY }}:${{ github.sha }}"' task-definition.json > temp.json && mv temp.json task-definition.json
-          cat task-definition.json  # 디버깅용 출력
+          # 현재 실행 중인 작업 수 확인
+          RUNNING_COUNT=$(aws ecs describe-services \
+            --cluster ${{ env.ECS_CLUSTER }} \
+            --services ${{ env.ECS_SERVICE }} \
+            --query 'services[0].runningCount' \
+            --output text)
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)